src/ThreeEditor/js/Viewport.js

```diff
@@ -3,15 +3,29 @@ import { TransformControls } from 'three/examples/jsm/controls/TransformControls
 import { SetPositionCommand } from './commands/SetPositionCommand';
 import { SetRotationCommand } from './commands/SetRotationCommand';
 import { SetScaleCommand } from './commands/SetScaleCommand';
-import { EditorControls } from './EditorControls';
 
 import { ViewportCamera } from './Viewport.Camera.js';
 import { ViewportInfo } from './Viewport.Info.js';
 
 import { UIPanel } from "./libs/ui";
 import { ViewHelper } from './Viewport.ViewHelper';
+import { EditorOrbitControls } from './EditorOrbitControls';
 
-export function Viewport(name, editor, { objects, grid, selectionBox }, cameraPosition) {
+import { GUI } from 'three/examples//jsm/libs/dat.gui.module.js';
+
+/*
+ * gridRotation = new THREE.Euler(0, Math.PI / 2, 0):
```

Comment:
The comment is to far away from the actual code.
Also - copying commented code into comment is a bad practice.
Couldn't we simply, for purpose of code readability extract this Euler angle object into separate variable, comment it properly and later use ?

Reply:
It is right above the constructor, so I don't think is too far.

Reply:
See [comment](https://github.com/yaptide/ui/pull/55#discussion_r707266665)